src/Facebook/FacebookBatchRequest.php

```diff
@@ -250,7 +257,10 @@ public function requestEntityToBatchArray(FacebookRequest $request, $requestName
             $batch['attached_files'] = $attachedFiles;
         }
 
-        // @TODO Add support for "omit_response_on_success"
+        if (isset($options['omit_response_on_success'])) {
```

Comment:
I'm thinking we just merge the options straight in so we can support all the existing options plus future ones.

```php
if (!empty($options)) {
    $batch = array_merge($options, $batch);
}
```

...or `array_merge($batch, $options)`, but that would allow the users to break things if they passed in an option named `headers` with random stuff in it for example.

But this makes me think we could get rid of `$requestName` and just pass in `name` through `$options`. Hum..

@yguedidi thoughts? :)